CMakeLists.txt

```diff
@@ -66,6 +66,23 @@ set(NBL_BUILTIN_RESOURCES_DIRECTORY_PATH "${NBL_ROOT_PATH}/include/nbl/builtin")
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${NBL_ROOT_PATH}/cmake" CACHE PATH "")
 get_property(NBL_IS_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
 
+#import .vsconfig file with VS components required for Nabla
+if(WIN32 AND MSVC)
+	option(NBL_IMPORT_VS_CONFIG "" ON)
```

Comment:
wrap the code into the function - have `NBL_IMPORT_VS_CONFIG` in `cmake/common.cmake` & make it 

```cmake
if(WIN32 AND "${CMAKE_GENERATOR}" MATCHES "Visual Studio")
  NBL_IMPORT_VS_CONFIG()
endif()
```

the problem with `MSVC` check is this way you look for the concretty compiler but a CMake invocation can have any compiler hooked with a VS generator & thats fine